jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-backend/src/test/java/org/jbpm/workbench/cm/backend/server/RemoteCaseManagementServiceImplTest.java

```diff
@@ -326,4 +329,52 @@ private CaseMilestone createTestMilestone(String caseMilestoneId, String caseMil
 
         return milestone;
     }
+
+    private TaskSummary createTaskSummary(Long taskId, String name, String status) {
+        TaskSummary taskSummary = TaskSummary.builder()
+                .id(taskId)
+                .name(name)
+                .status(status)
+                .createdOn(new Date())
+                .build();
+
+        return taskSummary;
+    }
+
+    private void assertActionEquals(TaskSummary taskSummary, CaseActionSummary caseActionSummary) {
+        assertNotNull(taskSummary);
+        assertNotNull(caseActionSummary);
+        assertEquals(taskSummary.getId(), caseActionSummary.getId());
+        assertEquals(taskSummary.getName(), caseActionSummary.getName());
+        assertEquals(taskSummary.getStatus(), caseActionSummary.getStatus());
+    }
+
+    @Test
+    public void getCaseActionsTest() {
```

Comment:
This test doesn't test the processing of return value of client.getAdHocFragments(containerId, caseId), but rather just that it was called. i.e. the following segment is not tested:
availableActions.addAll(client.getAdHocFragments(containerId, caseId)
        .stream()
        .map(new CaseActionAdhocMapper())
        .sorted(comparing(CaseActionSummary::getName)).collect(toList()));

Maybe writing one more test which will also have some sample AdHoc Fragments or test with only AdHoc Fragments?